internal/abi/prim/prim.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // Usize is an unsigned integer who's size is based on the system architecture.
-type Usize uint
+type Usize uint32
```

Comment:
This type probably doesn't make sense anymore.

Reply:
My plan more or less was to use this PR for getting the compatibility for Go 1.21, and leave any cleanups of the auto-generated code until later.

Reply:
Using this type in the definition of `Wstring` now.